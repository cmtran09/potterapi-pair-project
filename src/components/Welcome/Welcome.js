import React from "react"
import { Link } from "react-router-dom"

const Welcome = () => {
    return (
        <section className="hero welcome is-fullheight">
            <div className="hero-body">
                <div className="container">
                    <p className="title has-text-centered title is-2 has-text-white has-text-weight-bold">Potter World</p>
                    <p className="subtitle has-text-centered has-text-white">A place for Potterheads</p>
                    <Link to="/house">
                        <div className="has-text-centered">
                            <button className="button is-black center">
                                Enter
                        </button>
                        </div>

                    </Link>
                </div>
            </div>
        </section>
    )
}

export default Welcome 