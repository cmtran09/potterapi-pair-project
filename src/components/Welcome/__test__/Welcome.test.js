import React from "react"
import Welcome from "../Welcome"
import Enzyme, { shallow } from "enzyme"
import Adapter from "enzyme-adapter-react-16"
import toJSON from "enzyme-to-json"

Enzyme.configure({ adapter: new Adapter() })

test("matches snapshot", () => {
    const wrapper = shallow(<Welcome />)
    expect(toJSON(wrapper)).toMatchSnapshot()
})
test("button to have text Enter", () => {
    const wrapper = shallow(<Welcome />)
    const buttonText = wrapper.find("button").text()

    expect(buttonText).toBe("Enter")
})

// describe("welcome component",()=>{
//     test("it procceeds to correct link when clicked")
// })