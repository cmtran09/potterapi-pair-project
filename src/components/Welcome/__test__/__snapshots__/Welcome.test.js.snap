// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<section
  className="hero welcome is-fullheight"
>
  <div
    className="hero-body"
  >
    <div
      className="container"
    >
      <p
        className="title has-text-centered title is-2 has-text-white has-text-weight-bold"
      >
        Potter World
      </p>
      <p
        className="subtitle has-text-centered has-text-white"
      >
        A place for Potterheads
      </p>
      <Link
        to="/house"
      >
        <div
          className="has-text-centered"
        >
          <button
            className="button is-black center"
          >
            Enter
          </button>
        </div>
      </Link>
    </div>
  </div>
</section>
`;
